enterprise/internal/database/perms_store_test.go

```diff
@@ -41,23 +41,22 @@ func cleanupPermsTables(t *testing.T, s *permsStore) {
 	}
 }
 
-func bitmapToArray(bm *roaring.Bitmap) []int {
-	if bm == nil {
-		return []int{}
+func mapsetToArray(ms map[int32]struct{}) []int {
+	ints := []int{}
+	for id := range ms {
+		ints = append(ints, int(id))
 	}
+	sort.Slice(ints, func(i, j int) bool { return ints[i] < ints[j] })
 
-	uint32s := bm.ToArray()
-	ints := make([]int, len(uint32s))
-	for i := range uint32s {
-		ints[i] = int(uint32s[i])
-	}
 	return ints
 }
 
-func toBitmap(ids ...uint32) *roaring.Bitmap {
-	bm := roaring.NewBitmap()
-	bm.AddMany(ids)
-	return bm
+func toMapSet(ids ...int32) map[int32]struct{} {
```

Comment:
The case here seems being inconsistent with `mapsetToArray`
```suggestion
func toMapset(ids ...int32) map[int32]struct{} {
```

Or vice versa.